src/Middleware/LazyLoadingMiddleware.php

```diff
@@ -26,4 +26,11 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
         $middleware = $this->container->get($this->middlewareName);
         return $middleware->process($request, $handler);
     }
+    
+    /**
+     * For inspection of the lazy loaded middleware
+     */
+    public function getMiddlewareName(): string {
```

Comment:
Can we remove this method in favor of making the property public?

This would reduce the amount of changes once we are enforcing proper psalm types such as `class-string<MiddlewareInterface>`.

Reply:
Okay I've updated my PR and made it a public readonly property :)